Jenkinsfile

```diff
@@ -5,8 +5,9 @@ node {
     def branchName = env.BRANCH_NAME
     def tagName = branchName != "master" ? branchName : "latest"
     def imageName = repositoryName + ":" + tagName.replaceAll("[.:_/\\\\]", '-')
+    def testImageName = repositoryName + "-test" + ":" + tagName.replaceAll("[.:_/\\\\]", '-')
     def projectName = imageName.replaceAll("[.:_/\\\\]", '-')
-    def baseUrl = '/toolkit'
+    def baseUrl = '/' + tagName.replaceAll("[.:_/\\\\]", '-')
```

Comment:
Isn't this going to break the `master` build, that would now have a baseUrl `/latest` instead of `/toolkit`?